### PR TITLE
Prep 0.2.0 Release

### PR DESCRIPTION
**Pull request by ahuang11**
## Summary

Tested with:
image

## Checklist
- [ ] Summarized PR's changes in [CHANGELOG.md](https://github.com/PrefectHQ/prefect-snowflake/blob/main/CHANGELOG.md)

### CODE REVIEW

**Comment by ahuang11** [comment]
```
from prefect import flow
from prefect_snowflake.database import SnowflakeConnector, snowflake_query


@flow
def snowflake_query_flow():
    snowflake_connector = SnowflakeConnector.load("my-block")
    result = snowflake_query(
        "SELECT * FROM DAILY_FORECAST;",
        snowflake_connector,
        params={"id_param": 1}
    )
    return result

snowflake_query_flow()
```
image

tried out with UI too

File: prefect_snowflake/database.py [hunk 1]
```diff
@@ -42,9 +42,7 @@ class SnowflakeConnector(Block):
     schema_: str = Field(
         alias="schema", description="The name of the default schema to use"
     )
-    credentials: SnowflakeCredentials = Field(
-        ..., description="The credentials to authenticate with Snowflake"
-    )
+    credentials: SnowflakeCredentials
```
Comment (desertaxle) [inline prefect_snowflake/database.py:45]: Removing the field is a work around for some UI behavior, right?

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+### Deprecated
+
+### Removed
+
+### Fixed
+
+### Security
+
+## 0.2.0
+Released on August 12th, 2022.
```
Comment (desertaxle) [inline CHANGELOG.md:23]:
```suggestion
Released on August 15th, 2022.
```

Comment (desertaxle) [reply CHANGELOG.md:23]: Can we also add a line describing the breaking change that we made and what code updates need to be made?

File: prefect_snowflake/database.py (hunk 1 above)
Comment (ahuang11) [reply prefect_snowflake/database.py:45]: Yes, with Field(), it yields a JSON fields.

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,26 +8,57 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
 ### Security
-- Fixed revealing the input password nested under `connect_params` when logging `SnowflakeCredentials` - [#24](https://github.com/PrefectHQ/prefect-snowflake/pull/24)
 
 ## 0.2.0
-Released on August ??th, 2022.
+Released on August 15th, 2022.
+
+Note, with this release, fields from `SnowflakeCredentials` are migrated to `SnowflakeConnector`:
+```
```
Comment (desertaxle) [inline CHANGELOG.md:26]: I'd like to explicitly call out the fields, but I'm less sure about the breaking changes header.
```suggestion
### Breaking Changes

Note, with this release, the `database`, `warehouse` and `schema` fields from `SnowflakeCredentials` have been migrated to `SnowflakeConnector`:
```
```

**Review by desertaxle — CHANGES_REQUESTED** [review]

**Review by desertaxle — APPROVED** [review]

**Pull request merged**